### PR TITLE
Create rule4.py

### PR DESCRIPTION
**Pull request by botuser-jfit-rules**

### CODE REVIEW

**Comment by ayushgithub** [comment]
Can one of the admins verify this patch?

**Comment by botuser-jfit-rules** [comment]
ok to test

**Comment by ayushgithub** [comment]
ok to test

File: rules/rule4.py [hunk 1]
```diff
@@ -0,0 +1 @@
+assert (1 == 2), 'this is a test that fails'
```
Comment (ayushgithub) [inline rules/rule4.py]: Traceback (most recent call last):
  File "rules/rule4.py", line 1, in <module>
    assert (1 == 2), 'this is a test that fails'
AssertionError: this is a test that fails

File: rules/rule4.py [hunk 2]
```diff
@@ -0,0 +1 @@
+assert (1 == 3), 'this is a test that fails'
```
Comment (ayushgithub) [inline rules/rule4.py]: Traceback (most recent call last):
  File "rules/rule4.py", line 1, in <module>
    assert (1 == 3), 'this is a test that fails'
AssertionError: this is a test that fails

**Pull request merged**